metaseq/models/transformer_lm.py

```diff
@@ -96,28 +96,6 @@ class TransformerLanguageModelConfig(MetaseqDataclass):
             )
         },
     )
-    use_stable_embedding: Optional[bool] = field(
```

Comment:
We may yet regret this

Reply:
Do you want to turn it on in the next run?